src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -26,28 +27,15 @@ public static void SetPercentile(this IParameter parameter, double percentile)
          distributedParameter.Percentile = percentile;
       }
 
-      public static bool IsGlobalExpression(this IParameter parameter)
-      {
-         if (parameter == null)
-            return false;
 
-         return AllGlobalRelExpParameters.Contains(parameter.Name);
-      }
+
 
       public static bool IsIndividualMoleculeGlobal(this IParameter parameter) =>
          CoreConstants.Parameters.AllGlobalMoleculeParameters.Contains(parameter.Name);
 
-      public static bool IsExpression(this IParameter parameter)
-      {
-         if (parameter == null)
-            return false;
-
-         return parameter.IsGlobalExpression() || parameter.IsNamed(REL_EXP);
-      }
-
       public static bool IsExpressionOrOntogenyFactor(this IParameter parameter)
       {
-         if (parameter.IsExpression())
+         if (parameter.HasExpressionName())
```

Comment:
I'd like this to be as it was before. Move the extension method, make it use an IWithName but don't rename it 
Or remove HasExpreesionName. This doesn't read like a sentence

Reply:
If it's named IsExpression then I orefer it is on iparameter rather than Iwithname. Just because an iwithname has the appropriate name doesn't make it an expression parameter. I'll keep the ones as-is and put them back for Iparameter